Dockerfile

```diff
@@ -25,4 +21,4 @@ RUN bundle exec bootsnap precompile --gemfile app/ lib/
 RUN SECRET_KEY_BASE_DUMMY=1 bundle exec rails assets:precompile
 
 EXPOSE 3000
-CMD ["./bin/rails", "server"]
\ No newline at end of file
+CMD ["./bin/rails", "server", "--binding=0.0.0.0"]
```

Comment:
The explicit host binding is needed when running the server in the `development` Rails env.